README.md

```diff
@@ -1,49 +1,4 @@
-<p align="center"><a href="https://paystack.com/docs/libraries-and-plugins/plugins#others"><img src="./pix/banner.png?raw=true" alt="Paystack Add-On For Moodle"></a></p>
+# [Deprecated] Paystack Enrolment Plugin
+This plugin has been deprecated and will no longer be updated. 
```

Comment:
Can we add "or supported"? In case anyone is currently using it and reports any bugs to us, we've clearly stated that we won't support/fix their issues